src/test/java/com/gooddata/http/client/TestUtils.java

```diff
@@ -67,9 +67,9 @@ static void logout(HttpClient client, HttpHost httpHost, String profile, int exp
         }
     }
 
-    static HttpClient createGoodDataClient(String login, String password, HttpHost host) {
+    static HttpClient createGoodDataClient(String login, String password, HttpHost host, String gdcHostHeader) {
         final HttpClient httpClient = HttpClientBuilder.create().build();
-        final SSTRetrievalStrategy sstStrategy = new LoginSSTRetrievalStrategy(login, password);
-        return new GoodDataHttpClient(httpClient, host, sstStrategy);
+        final SSTRetrievalStrategy sstStrategy = new LoginSSTRetrievalStrategy(login, password, gdcHostHeader);
+        return new GoodDataHttpClient(httpClient, host, sstStrategy, gdcHostHeader);
```

Comment:
The usage is not very friendly (passing the value to strategy and then to client). Consider simplification to use it only once.